go/stellar/stellar.go

```diff
@@ -47,7 +62,10 @@ func CreateWallet(ctx context.Context, g *libkb.GlobalContext) (created bool, er
 	default:
 		return false, err
 	}
-	return true, err
+	if err := setNewWalletDisplayCurrency(ctx, g, clearBundle); err != nil {
+		g.Log.CWarningf(ctx, "Failed to set default currency for primary account: %s", err)
+	}
```

Comment:
Why do this on the client? If it's done on the server it can be guaranteed to succeed and we can change it whenever.